src/dbnode/storage/bootstrap/bootstrapper/peers/source.go

```diff
@@ -277,10 +250,7 @@ func (s *peersSource) readData(
 		concurrency = s.opts.ShardPersistenceConcurrency()
 	}
 
-	s.log.Info("peers bootstrapper bootstrapping shards for ranges",
-		zap.Int("shards", count),
-		zap.Int("concurrency", concurrency),
-		zap.Bool("shouldPersist", shouldPersist))
+	instrCtx := s.instrumentation.bootstrapShardsStarted(count, concurrency, shouldPersist)
```

Comment:
Should use a `defer instrCtx.bootstrapShardsCompleted()` here to capture if an early return with an occurs?